model/src/olivia.rs

```diff
@@ -74,10 +75,10 @@ impl BitMexPriceEventId {
     /// Checks whether this event has likely already occurred.
     ///
     /// We can't be sure about it because our local clock might be off from the oracle's clock.
-    pub fn has_likely_occured(&self) -> bool {
+    pub fn has_likely_occurred(&self) -> bool {
         let now = OffsetDateTime::now_utc();
 
-        now > self.timestamp
+        now > self.timestamp + Duration::minutes(1)
```

Comment:
🙃 You can also use this, which is a tiny bit more readable:

```suggestion
        use time::ext::NumericalDuration;
        now > self.timestamp + 1.minutes()
```